modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -612,6 +612,38 @@
             end
           end
         end
+
+        context 'it has an errors array' do
+          let(:claim) do
+            create(
+              :auto_established_claim_with_supporting_documents,
+              :status_errored,
+              source: 'abraham lincoln',
+              evss_response: [
+                {
+                  severity: 'ERROR',
+                  detail: 'Something happened',
+                  key: 'test.path.here'
+                }
+              ]
+            )
+          end
+          let(:claim_by_id_path) { "/services/claims/v2/veterans/#{veteran_id}/claims/#{claim.id}" }
+
+          it "returns a claim with the 'claimId' and 'lighthouseId' set" do
```

Comment:
Spec name should reflect what the spec is testing.

Something like, `it "returns a claim with the 'errors' attribute populated" do`.